lib/components/table.vue

```diff
@@ -441,7 +438,18 @@
                 }
 
                 // Shallow copy of items, so we don't mutate the original array order/size
-                items = items.slice();
+
+                items = JSON.parse(JSON.stringify(items))
+
+                // Apply formatter only if fields object defined
+                if (this.fields && this.fields.toString() === '[object Object]'){
+                    const keysToFormat = keys(this.fields).filter((k, i, a) => this.hasFormatter(this.fields[k]))
+                    keysToFormat.forEach(k =>{
+                        items.forEach(item =>{
+                            item[k] = this.callFormatter(item, k, this.fields[k])
+                        }, this)
```

Comment:
So I see why using JSON.stringy now... since you are altering the raw data, rather than at the render level (in the template)